aztk/client.py

```diff
@@ -61,6 +61,9 @@ def __delete_pool_and_job(self, pool_id: str):
 
         if pool_exists:
             self.batch_client.pool.delete(pool_id)
+        
+        cluster_data = self._get_cluster_data(pool_id)
+        cluster_data.delete_container(pool_id)
```

Comment:
sweet!

Reply:
yeah this was a super simple change because of the `cluster_data` refactor